src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -314,6 +325,16 @@ class MultiDiracDeterminant : public WaveFunctionComponent
                                           const std::vector<std::pair<int, int>>& pairs,
                                           const std::vector<RealType>& sign);
 
+  void mw_BuildDotProductsAndCalculateRatios(int nw,
```

Comment:
No documentation around both single and multi-walker functions. It is worth adding some documentation.